.github/actions/integration-tests/setup-ci-environment

```diff
@@ -101,7 +101,10 @@ case "${CI_ENDPOINT}" in
         update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
 
         snap install microk8s --classic
-        microk8s status --wait-ready
+        if ! microk8s status --wait-ready; then
```

Comment:
--wait-ready implies in timeout if not ready ? just make sure this command returns back and doesn't get polling forever

Reply:
I'm not exactly sure what the timeout behavior of `--wait-ready` is.  However, the job itself will timeout eventually if no progress is made so it is impossible for the job to run forever.

The waiting logic here is unchanged, what is changed is checking for the return code of that command.  This logic has been working fine for a very long time, but what I recently observed was that the `snap install` failed for some reason and since we weren't checking anything here then the job ran for much longer before it failed due to microk8s missing.  I decided to improve this by adding a simple check here for the return status of the `--wait-ready` (which will fail if microk8s installation fails).  I could have added a check to the `snap install` command but I figured that a single check on the `--wait-ready` would handle a lot more failure cases (install failure, init failure, etc.).